crates/bevy_pbr/src/prepass/mod.rs

```diff
@@ -268,20 +268,22 @@ type PreviousMeshFilter = Or<(With<Mesh3d>, With<MeshletMesh3d>)>;
 pub fn update_mesh_previous_global_transforms(
     mut commands: Commands,
     views: Query<&Camera, Or<(With<Camera3d>, With<ShadowView>)>>,
-    meshes: Query<(Entity, &GlobalTransform, Option<&PreviousGlobalTransform>), PreviousMeshFilter>,
+    new_meshes: Query<
```

Comment:
`new_meshes` can use just `With<Mesh3d>`, as MeshletMesh3d requires PreviousGlobalTransform.

Reply:
It would be a bit brittle to make this code rely on the behavior of another component's required components. I'd rather just be explicit here: if something doesn't have that component, I need to add it.

Reply:
Plus, as that filter type changes, this query will update to follow it, adding an exception is a place that could break in the future. As is, there is no problem here, `MeshletMesh3d` currently requiring the `PreviousGlobalTransform` just means it won't be included in the query, there is no negative perf impact.